content/en/docs/quick-start/all-in-one-on-linux.md

```diff
@@ -186,3 +186,6 @@ After logging in the console, you can check the status of different components i
 ## Enable Pluggable Components (Optional)
 
 The guide above is used only for minimal installation by default. To enable other components in KubeSphere, see [Enable Pluggable Components](../../pluggable-components/) for more details.
+
+## Demo
+[![asciicast](https://asciinema.org/a/362291.png)](https://asciinema.org/a/362291)
```

Comment:
@ruiyaoOps  Please embed the player instead of the image.
![image](https://user-images.githubusercontent.com/40452856/94515803-fd986c80-0256-11eb-9688-6ed35c75e4ed.png)